device/iot-device-samples/multiplexing-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/MultiplexClientManager.java

```diff
@@ -0,0 +1,47 @@
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.device.IotHubConnectionStatusChangeCallback;
+import com.microsoft.azure.sdk.iot.device.MultiplexingClient;
+import lombok.experimental.Delegate;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class MultiplexClientManager extends ClientManagerBase {
```

Comment:
Since this class is in the samples folder, I am guessing this class (as well as the other managers) are intended as helper classes for the samples, correct?
It would be nice to have class documentation on what this class is, how it's intended to be used, and what benefits one would gain from using such helper classes.